docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,360 @@
+* SSID: The "SSID" field defines which SSID will be configured on the device using the authentication profile.
+* EAP-Type: The EAP type defines the authentication method supported and should be set to EAP-TLS to integrate with the PacketFence PKI.
+* Security type: The security type should be set to WPA2 to integrate with the PacketFence PKI.
+* PKI Provider: This should match the provider you configured earlier in the PKI provider section.
+
+The following is an example on how to configure an EAP-TLS connection for Windows/Android/Mac OS X/iOS
+
+image::docs/images/scep-ms-pki-eaptls-example.png[scaledwidth="100%",alt="Provisioner EAP-TLS configuration"]
+
+Mac OS X/iOS require the provisioning profile to be signed if you want to remove the `untrusted` warning when installing the profile. You need to sign it with a Certification Authority already trusted by the device such as e.g. VeriSign.
+Configuring this has to be done in the 'Signing' tab in the "Apple devices". 
+
+image::docs/images/packetfence-pki-eaptls-sign-example.png[scaledwidth="100%",alt="Signing provisioner"]
+
+Fill out the fields with the contents of the Base64 encoded certificates.
+To extract this information from a pem formatted certificate, copy the file content included between the begin and end tag, not including the delimiters themselves.
+
+Certificate file example:
+
+----
+----- BEGIN CERTIFICATE -----
+1234567890asdfghjkl
+zxcvbnmqwertyuiop78
+----- END CERTIFICATE -----
+----
+
+Copy everything between the BEGIN and END lines, but not the lines themselves.
+Repeat this operation for the certificate key and intermediate certificate.
+
+----
+----- BEGIN PRIVATE KEY -----
+1234567890asdfghjkl
+zxcvbnmqwertyuiop78
+----- END PRIVATE KEY -----
+----
+
+Portal Profiles Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Provisioners have to be enabled on the Portal Profiles configuration in the PacketFence GUI.
+
+Under `Configuration -> Main -> Portal Profiles`, select each of the provisioners created above which should be active for the profile.
+If no portal profile is defined, configure the "default" profile to use the provisioners created.
+
+Passthroughs required for Android
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Android devices require passthroughs to be created to allow them to fetch the configuration application from the Google Play Store.
+
+IMPORTANT: Passthroughs will vary depending on the location where your Google account was created. You will need to add some extra passthroughs for the store of your country. In the section debug there is a how-to determine which address you need to add.
+
+Add the following to the "Trapping" section of the Configuration tab in the PacketFence GUI.
+
+----
+passthrough=enabled
+passthroughs=*.ggpht.com,*.googleusercontent.com,android.clients.google.com,
+  *.googleapis.com,*.android.clients.google.com,*.gvt1.com
+----
+
+Step 3: Appendix
```

Comment:
Appendix is not exactly what I would call a step.

And this is a section so it's '---------------' not '~~~~~~~~~~~~'